src/plugins/data_view_field_editor/public/components/flyout_panels/flyout_panels.scss

```diff
@@ -33,6 +33,7 @@
 
   &__header {
     padding: 0 !important;
+    overflow-wrap: break-word;
```

Comment:
For better BWC with other browsers, I'd recommend either applying the [EUI utility class](https://elastic.github.io/eui/#/utilities/css-utility-classes#text) `.eui-textBreakWord` on the element or the Sass mixin equivelant: `@include euiTextBreakWord;` so that it only breaks if it **has** to.

Reply:
via this commit: https://github.com/elastic/kibana/pull/122690/commits/517266fe5966fdf5c8641a5fe9a7a163d2d88f96

changing

```
overflow-wrap: break-word;
```

to

```
@include euiTextBreakWord;
```

updated the styles applied to the element from (via `overflow-wrap: break-word;`):

![before-mixin](https://user-images.githubusercontent.com/4459398/149024820-82ef8ae7-84d1-4ca8-99d9-e297c8464eef.png)

to (via `@include euiTextBreakWord;`):

![after-mixin](https://user-images.githubusercontent.com/4459398/149024851-661c1bb2-3245-4ee0-929c-57a08164acaf.png)